apprentice_teacher.rb

```diff
@@ -1,21 +1,10 @@
-class ApprenticeTeacher
-  attr_reader :age, :salary, :phase, :target_raise
-  attr_accessor :name
+require_relative 'faculty'
+class ApprenticeTeacher < Faculty
 
   def initialize(options={})
-    @age = options.fetch(:age, 0)
-    @name = options.fetch(:name, "")
+    super
     @target_raise = 800
-    @phase = 3
-  end
-
-  def offer_high_five
-    "High five!"
-  end
-
-  def set_phase(num)
-    @phase = num
-    "Cool, I've always wanted to teach phase #{num}!"
+    @min_rating = 80
   end
 
   def teach_stuff
```

Comment:
this method is in two classes right now. how can you refactor this so the method is set in one place instead of two?